README.md

```diff
@@ -145,7 +174,8 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+    Run the code and input commands; end each command with a period "."
+    input "q" to stop.
```

Comment:
It took me a while to work out that the input had to end "EOI." :)

# 8

Two problems here. First, the question asked you top translate the table into your programming language. I was expecting to see a data structure, but instead you implement the FSM in code.

Secondly, there are no tests.